contrib/fixclock-hook

```diff
@@ -26,27 +28,96 @@ You can change the NTP server like this:
 Unfortunately, something went wrong...
 
 $ERROR
+"""
+
+LXC_TPL = f"""\
```

Comment:
should it be called `LXC_TPL` if any container type is detected?

Reply:
Noticed that myself. I'll change it.